nitro-protocol/gas-benchmarks/benchmark.test.ts

```diff
@@ -199,72 +205,100 @@ describe('Consumes the expected gas for sad-path exits', () => {
     ).toEqual(gasRequiredTo.ETHexitSadLedgerFunded.satp.total);
   });
 
-  // TODO unskip this test when the contracts are satp compatible
-  it.skip(`when exiting a virtual funded (with ETH) channel`, async () => {
+  it(`when exiting a virtual funded (with ETH) channel`, async () => {
```

Comment:
This is a good place to start reviewing. From here to the end of the file you can see the operations Bob is doing to get his money.